CoroutineTemplate/app/build.gradle.kts

```diff
@@ -39,12 +46,13 @@ android {
             isDebuggable = false
             isShrinkResources = true
             proguardFiles(getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro")
-            signingConfig = signingConfigs["config"]
+            signingConfig = signingConfigs[BuildType.RELEASE]
             buildConfigField("String", "BASE_API_URL", "\"https://jsonplaceholder.typicode.com/\"")
         }
         getByName(BuildType.DEBUG) {
```

Comment:
Can we have a line break here before `getByName(BuildType.DEBUG) {` to make it easy to find a buildType?

Reply:
@lydiasama not a big deal, added